File: 1608. Special Array With X Elements Greater Than or Equal X.go
func specialArray(nums []int) int {
    for i := 0; i <= len(nums); i++ {
        count := 0
        for j := 0; j < len(nums); j++ {
            if nums[j] >= i {
                count++
            }
        }
        if count == i {
            return i
        }
    }
    return -1
}

File: 1608. Special Array With X Elements Greater Than or Equal X.py
class Solution:
    def specialArray(self, nums: List[int]) -> int:
        for i in range(len(nums)+1):
            x = i
            count = 0
            for num in nums:
                if num >= x:
                    count += 1
            if count == x:
                return x
        return -1
       

